input/chainsync/tx.go

```diff
@@ -26,19 +27,27 @@ type TransactionEvent struct {
 	TransactionCbor byteSliceJsonHex           `json:"transactionCbor,omitempty"`
 	Inputs          []ledger.TransactionInput  `json:"inputs"`
 	Outputs         []ledger.TransactionOutput `json:"outputs"`
+	Metadata        *cbor.Value                `json:"metadata,omitempty"`
+	MetadataCbor    byteSliceJsonHex           `json:"metadataCbor,omitempty"`
```

Comment:
I think the `metadataCbor` field is redundant here, since the `metadata` output will also contain the original CBOR